sympy/stats/tests/test_joint_rv.py

```diff
@@ -41,8 +50,8 @@ def test_MultivariateTDist():
 def test_multivariate_laplace():
     from sympy.stats.crv_types import Laplace
     raises(ValueError, lambda: Laplace('T', [1, 2], [[1, 2], [2, 1]]))
-    L = Laplace('L', [1, 0], [[1, 2], [0, 1]])
-    assert density(L)(2, 3) == exp(2)*besselk(0, sqrt(3))/pi
+    L = Laplace('L', [1, 0], [[1, 0], [0, 1]])
+    assert density(L)(2, 3) == exp(2)*besselk(0, sqrt(39))/pi
```

Comment:
Can you retain both tests?

Reply:
@sidhantnagpal the deleted test is incorrect since `[[1, 2], [0, 1]]` is not `positive semidefinite`. A matrix is `positive semidefinite` only if it is symmetric.

Reply:
Does the former raise an error to account for that?

Reply:
No, actually we were just checking for eigenvalues to be positive for psd where as `is_positive_definite` checks for positive eigenvalues and symmetry. I asked this on gitter and got the reply that `is_positive_definite` is giving the correct answer.